content/1.3-migration.md

```diff
@@ -89,6 +91,40 @@ Along side some of the breaking mobile changes [listed above](#breaking-mobile),
 
 - The plugin now allows for local file access on both iOS and Android. You can construct file system URLs manually (`http://localhost:<port>/local-filesystem/<path>`) or use `WebAppLocalServer.localFileSystemUrl()` to convert a `file://` URL. On iOS, core plugins like `cordova-plugin-file` and `cordova-plugin-camera` (using `Camera.DestinationType.FILE_URI`) will now automatically return these URLs instead of `file://` URLs.
 
+<h2 id="react">Recommended changes: React</h2>
+
+As mentioned in the [breaking changes](#breaking-changes), the React Atmosphere package now "peer" depends on the React npm packages (see the [instructions from package authors](#packages) above for more details on this). If you are using the `react` Atmosphere package (and thus the `react-runtime` package), this means you are required to `npm install` the following set of NPM packages into your app (read the [Using Packages](using-packages.html) article for more details about how to manage your NPM dependencies):
```

Comment:
Good question. I think technically it's the same thing as a peer dependency---in both cases your code will not run if you do not install the peer dependency--- it's just that in NPM actually enforces that for you, and helps with resolving dependencies (maybe?), whereas here the package craps out with an error. 

So (from the user's perspective) it's effectively the same thing. What would you call it?